.github/workflows/release.yml

```diff
@@ -59,3 +59,5 @@ jobs:
       with:
         files: |
           out/charts/kmesh-helm-${{ github.ref_name }}.tgz
+    - name: Push kmesh helm package
+      run: make helm-push
```

Comment:
This is not enough, we need to  make sure it has logined in

Reply:
- name: Login to Github Container Registry
        run: echo "${{ secrets.GH_PAT }}" | docker login ghcr.io -u $ --password-stdin